sound/soc/sof/topology.c

```diff
@@ -1759,6 +1759,9 @@ static int sof_widget_load_pipeline(struct snd_soc_component *scomp, int index,
 		goto err;
 	}
 
+	if (sof_core_debug & SOF_DBG_DISABLE_MULTICORE)
+		pipeline->core = 0;
```

Comment:
For consistency, this should be SOF_DSP_PRIMARY_CORE. I think it's ok to talk about Core0 in the commit as that's the reality now.

Reply:
ack, good feedback @kv2019i